app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -222,17 +226,17 @@ private fun ConversationScreenContent(
 @Composable
 private fun getSnackbarMessage(messageCode: ConversationSnackbarMessages): Pair<String, String?> {
     val msg = when (messageCode) {
-        is ErrorMaxAssetSize -> stringResource(R.string.error_conversation_max_asset_size_limit, messageCode.maxLimitInMB)
-        is ErrorMaxImageSize -> stringResource(R.string.error_conversation_max_image_size_limit)
-        is ErrorSendingImage -> stringResource(R.string.error_conversation_sending_image)
-        is ErrorSendingAsset -> stringResource(R.string.error_conversation_sending_asset)
-        is ErrorDownloadingAsset -> stringResource(R.string.error_conversation_downloading_asset)
-        is ErrorOpeningAssetFile -> stringResource(R.string.error_conversation_opening_asset_file)
         is OnFileDownloaded -> stringResource(R.string.conversation_on_file_downloaded, messageCode.assetName ?: "")
-        else -> stringResource(R.string.error_conversation_generic)
+        is ErrorMaxAssetSize -> stringResource(R.string.error_conversation_max_asset_size_limit, messageCode.maxLimitInMB)
+        ErrorMaxImageSize -> stringResource(R.string.error_conversation_max_image_size_limit)
```

Comment:
I'm not 100% sure, but I think when we are checking for instances, we should be using is (just like was before). Was there a particular reason for the change?

Reply:
On `when` clauses, when checking objects types we can ommit the `is` operator. However on classes we must add it. I thought it added a bit of visual differentiation to know which ones are classes also holding some internal values, but I can add `is` operator if you think it's cleaner that way 😅

Reply:
It makes sense. I think we don't have a convention for this as a team, so I'm fine either way =)